core/src/main/java/com/datastax/oss/driver/api/core/config/DefaultDriverOption.java

```diff
@@ -687,6 +687,12 @@
    * <p>Value-type: {@link java.time.Duration Duration}
    */
   REPREPARE_TIMEOUT("advanced.prepared-statements.reprepare-on-up.timeout"),
+  /**
+   * Whether the prepared statements cache use weak values.
+   *
+   * <p>Value-type: boolean
+   */
+  PREPARED_CACHE_WEAK_VALUES("advanced.prepared-statements.prepared-cache.weak-values"),
```

Comment:
You must add new enum constants at the end of the list, otherwise the `revapi` plugin will complain that this is a breaking change, and `mvn clean verify` will fail.

Reply:
OK, already updated